integration_tests/test_suites/dagster-azure-live-tests/integration_tests/test_compute_log_manager.py

```diff
@@ -57,3 +59,14 @@ def test_compute_log_manager(
 
     assert stdout.count("Printing without context") == 10
     assert stderr.count("Logging using context") == 10
+
+    assert logs_captured_data.stdout_uri_or_path
+    assert logs_captured_data.stderr_uri_or_path
+    assert logs_captured_data.external_stdout_url.endswith(logs_captured_data.stdout_uri_or_path)
```

Comment:
if possible, would be nice to actually test the uri download + download from the cli command.

Reply:
not a bad idea - installing the azure cli and configuring it programmatically with the test credentials shouldn't be too hard if it supports the same credentialing. would like to do so in a different PR on this stack though.

Reply:
Ah yea it's a good point that you'll need to install cli separately and that will probably be a pain. In my brain installing azure python would install cli but that very well might not be true.

Feel free to re request review and disregard